frontend/packages/knative-plugin/src/yaml-templates.ts

```diff
@@ -16,3 +16,5 @@ spec:
       - image: openshift/hello-openshift
 `,
 );
+
+export const defaultYamlTemplate = yamlTemplates.getIn([ServiceModel, 'default']);
```

Comment:
We don't need the `ImmutableMap` here anymore. We can export the template directly:
```suggestion
export const defaultServiceYamlTemplate = `...`
```

Reply:
fixed